modules/claims_api/spec/sidekiq/shared_reporting_examples_spec.rb

```diff
@@ -60,23 +60,21 @@
 
   it 'includes 526EZ claims from VaGov' do
     with_settings(Settings.claims_api, report_enabled: true) do
-      claim_one = FactoryBot.create(:auto_established_claim_va_gov, :errored, created_at: Time.zone.now,
-                                                                              transaction_id: '467384632186')
-      claim_two = FactoryBot.create(:auto_established_claim_va_gov, :errored, created_at: Time.zone.now,
+      claim_one = FactoryBot.create(:auto_established_claim_va_gov, :errored, created_at: 2.seconds.ago,
                                                                               transaction_id: '467384632187')
-      FactoryBot.create(:auto_established_claim_va_gov, :errored, created_at: Time.zone.now,
-                                                                  transaction_id: '467384632187')
+      claim_two = FactoryBot.create(:auto_established_claim_va_gov, :errored, created_at: 3.seconds.ago,
+                                                                              transaction_id: '467384632186')
 
       job = described_class.new
       job.perform
       va_gov_groups = job.unsuccessful_va_gov_claims_submissions
       first_group = va_gov_groups[1]
       first_transaction_id = claim_one[:transaction_id]
-      expect(first_group[0][:transaction_id]).to eq(first_transaction_id)
-
       second_group = va_gov_groups[2]
       second_transaction_id = claim_two[:transaction_id]
-      expect(second_group[0][:transaction_id]).to eq(second_transaction_id)
+
+      expect(second_group[0]).to include(transaction_id: second_transaction_id.to_s)
+      expect(first_group[0]).to include(transaction_id: first_transaction_id.to_s)
```

Comment:
Unfortunately this still came in out of order and without the expected values so it failed.

<img width="1009" alt="Screenshot 2024-10-31 at 4 23 31 PM" src="https://github.com/user-attachments/assets/921c5f06-e84e-4ce0-aa0a-bef0c9f7a568">

Is it enough to just verify each group has 1?

```
expect(first_group.count).to eq(1)
expect(second_group.count).to eq(1)
```

Since we just need to make they do not group together that would cover it I think.  Thoughts?